mne/coreg.py

```diff
@@ -1756,6 +1756,22 @@ def _setup_icp(self, n_scale_params):
             mri_pts *= self._scale  # not done in fit_matched_points
         return head_pts, mri_pts, weights
 
+    def set_fid_match(self, match):
+        """Set the strategy for fitting anatomical landmark (fiducial) points.
+
+        Parameters
+        ----------
+        match : 'nearest' | 'matched'
+            Alignment strategy; ``'nearest'`` aligns anatomical landmarks to
+            any point on the head surface; ``'matched'`` aligns to the fiducial
+            points in the MRI.
+        """
+        if match not in self._icp_fid_matches:
+            msg = (f'"match" must be one of {self._icp_fid_matches}, got '
+                   f'{match}')
+            raise ValueError(msg)
+        self._icp_fid_match = match
```

Comment:
can you make this function return self liek the other setters so we can chain them in one line?

Reply:
```suggestion
        self._icp_fid_match = match
        return self
```